ntbs-service/Migrations/NtbsContextModelSnapshot.cs

```diff
@@ -13992,7 +13992,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                                 .OnDelete(DeleteBehavior.Cascade);
                         });
 
-                    b.OwnsOne("ntbs_service.Models.Entities.Episode", "Episode", b1 =>
+                    b.OwnsOne("ntbs_service.Models.Entities.HospitalDetails", "HospitalDetails", b1 =>

```

Comment:
Haven't checked here, but I imagine the snapshot still has a hold of all the old index names? I think (might be worth checking) - that a manual modification of the names might be our only route with `sp_rename` being used.

Reply:
I have checked indexes they seem fine on snapshot